README.md

```diff
@@ -196,8 +196,6 @@ configuration:
 Shell aliases and scripts:
 
 * `b` for `bundle`.
-* `g` with no arguments is `git status` and with arguments acts like `git`.
-* `git-churn` to show churn for the files changed in the branch.
```

Comment:
This also exists: https://github.com/thoughtbot/dotfiles/blob/master/bin/git-churn